session/role_integ_test.go

```diff
@@ -329,7 +329,9 @@ func Test_Integ_Role(t *testing.T) {
 		err = bobCh.UnsubChUpdates()
 		assert.Error(t, err, "bob unsubscribing from channel updates after closing notification should error")
 		t.Log(err)
-		require.EqualError(t, err, perun.ErrChClosed.Error())
+
+		apiError := perun.NewAPIErrV2FailedPreCondition(session.ErrChClosed.Error())
+		require.Equal(t, err, apiError)
```

Comment:
Assert for the individual properties of the error as it has been done in other parts of this PR.